File: 23-09-2026/rms_value.py
import numpy as np

def rms(a, n):
    arr = np.array(a)
    
    sum_of_squares = np.dot(arr, arr)
    
    rms_value = np.sqrt(sum_of_squares / n)
    return rms_value

n = int(input().strip())
readings = list(map(float, input().strip().split()))

result = rms(readings, n)
print(f"{result:.2f}")

File: 23-09-2026/rms_value.c
#include<stdio.h>
#include<math.h>
double rms(double a[],int n)
{
double rms_Value = 0;
for(int i = 0;i<n;i++)
{
rms_Value += (a[i]*a[i])/n;


}
return sqrt(rms_Value);

}
int main()
{
int n = 0;
scanf("%d",&n);
double a[n];
for(int i=0;i<n;i++)
{
scanf("%lf",&a[i]);

}
printf("%f",rms(a,n));
}
